src/pages/Profile/Profile.vue

```diff
@@ -1,13 +1,110 @@
 <template>
-  <div>Profile</div>
+ 
+    <v-flex class="white elevation-2 fluid">
+        <v-toolbar flat dense class="blue darken-3" dark>
+        <v-toolbar-title>User Profile</v-toolbar-title>
+        </v-toolbar>
+        <div class="pl-4 pr-4 pt-3 pb-2">
+          <v-container grid >
+            <v-layout row wrap>
+            <v-flex md3 xs12 sm6>
+              <v-layout wrap>
+                <div>
+                <v-avatar
+                :tile="false"
```

Comment:
set this true for get squared profile picture

Reply:
circle is best for this design